cmd/talosctl/cmd/talos/netstat.go

```diff
@@ -210,6 +248,12 @@ func printNetstat(response *machine.NetstatResponse) error {
 				}
 			}
 
+			if netstatCmdFlags.pods {
+				args = append(args, []interface{}{
```

Comment:
why `[]interface` when args is a list of strings?

Reply:
sre you thinking of the args inside `cobra.command`?
`args` is indeed a `[]string` there.

This is a builds a `args[]` array for the tab writer:
initialized here:
https://github.com/siderolabs/talos/blob/ae339a2024c2c7d33b491dfee7e980be2674736a/cmd/talosctl/cmd/talos/netstat.go#L212

used here: 
https://github.com/siderolabs/talos/blob/ae339a2024c2c7d33b491dfee7e980be2674736a/cmd/talosctl/cmd/talos/netstat.go#L268
as Fprintf takes a mix of types depending on the format string